devicemodel/hw/platform/usb_pmapper.c

```diff
@@ -889,7 +889,7 @@ usb_dev_data(void *pdata, struct usb_xfer *xfer, int dir, int epctx)
 }
 
 static void
-clear_uas_desc(struct usb_dev *udev, uint8_t *data, uint32_t len)
+clear_uas_desc(struct usb_dev *udev, uint8_t *data, int len)
```

Comment:
Does this violate one of our project's C coding guidelines about not using basic numerical types? See:  https://projectacrn.github.io/latest/developer-guides/c_coding_guidelines.html?highlight=coding%20guideline#c-ty-26-the-basic-numerical-types-shall-not-be-used-other-than-in-typedefs